scripts/setup_udev_rules.sh

```diff
@@ -1,20 +1,53 @@
 #!/bin/bash -e
 
-echo "Setting-up permissions for RealSense devices"
+# USAGE:
+# Normal usecase - without any parameters.
+#
+# [optional parameters]:
+# --uninstall : remove permissions for realsense devices.
+
+install=true
+auto_power_off=false
+
+for var in "$@"
+do
+    if [ "$var" = "--uninstall" ]; then
+        install=false
+    fi
+done
+
+if [ "$install" = true ]; then
+    echo "Setting-up permissions for RealSense devices"
+    if [ "$auto_power_off" = true ]; then
+        echo "Setting-up RealSense Device auto power off."
+        sudo apt install -q=3 at || (echo "Failed to install package 'at'. Remove flag --auto_power_off and run again." && exit 1)
+    fi
+else
+    echo "Remove permissions for RealSense devices"
+fi
 
 exec 3>&2
 exec 2> /dev/null
 con_dev=$(ls /dev/video* | wc -l)
 exec 2>&3
 
-if [ $con_dev -ne 0 ];
-then
-	echo -e "\e[32m"
-	read -p "Remove all RealSense cameras attached. Hit any key when ready"
-	echo -e "\e[0m"
+if [ "$install" = true ]; then
+    if [ $con_dev -ne 0 ];
+    then
+        echo -e "\e[32m"
+        read -p "Remove all RealSense cameras attached. Hit any key when ready"
+        echo -e "\e[0m"
+    fi
+    sudo cp config/99-realsense-libusb.rules /etc/udev/rules.d/
+    echo | sudo tee -a /etc/udev/rules.d/99-realsense-libusb.rules > /dev/null
+    echo "KERNEL==\"iio*\", ATTRS{idVendor}==\"8086\", ATTRS{idProduct}==\"0ad5|0afe|0aff|0b00|0b01|0b3a|0b3d|0b64\", RUN+=\"/bin/sh -c '(major=\`uname -r | cut -d \\\".\\\" -f1\` && minor=\`uname -r | cut -d \\\".\\\" -f2\` && (([ \\\$major -eq 4 ] && [ \\\$minor -ge 15 ]) || [ \\\$major -ge 5 ])) &&  enfile=/sys/%p/buffer/enable && echo \\\"COUNTER=0; while [ \\\$COUNTER -lt 20 ] && grep -q 0 \$enfile; do sleep 0.01; COUNTER=\\\$((COUNTER+1)); done && echo 0 > \$enfile\\\" | at now'\"" | sudo tee -a /etc/udev/rules.d/99-realsense-libusb.rules > /dev/null
```

Comment:
Pls add in-place description for what is doing on for maintainers.
One more thing - currently it will run for both Accel and Gyro, but we know that the issue occurs with Accel.
so even if accelerometer will be fixed immediately we'll still wait for 200 msec for gyro ( that won't start)
Can the condition "KERNEL==\"iio*\", ATTRS{idVendor}==\"8086\" be further fine-tuned?